frontend/src/components/GifWidget/GifWidget.tsx

```diff
@@ -0,0 +1,84 @@
+import { SetStateAction, useEffect, useState } from "react";
+import {
+    GifWidgetStyled,
+    GifWidgetContent,
+    GifWidgetSearch,
+    GifList,
+    GifListImage,
+    GifWidgetAtrtibutionSection,
+    GiphyAttributionLogo
+} from "./GifWidget.styled";
+import axios from "axios";
+import GiphyAttributionLogoImage from "assets/GiphyAttributionLogo.png";
+
+const GifWidget = () => {
+    const [gifsLoading, setGifsLoading] = useState(true);
+    const [gifList, setGifList] = useState<any[]>([]);
+    const [gifSearchInput, setGifSearchInput] = useState("");
+
+    const findGif = (e: { target: { value: SetStateAction<string> } }) => {
+        setGifSearchInput(e.target.value);
+        setGifsLoading(true);
+    };
+
+    const getGifUrl = (fullGifUrl: string) => {
+        // image url which should be passed into chat box
+        console.log("image url", fullGifUrl);
+    };
```

Comment:
This code can be moved to file with utill/helper functions.

Reply:
this code is unnecessary now, i remove it.